mkdocs.yml

```diff
@@ -35,17 +35,7 @@ nav:
   - Learn about Elmish : 'component-elmish.md'
 - The SAFE Template:
   - Overview: 'template-overview.md'
-  - Run and Build the app: 'template-fake.md'
```

Comment:
How about just the following:
* Leave the files that have been removed
* Prefix them with "legacy-*" so that they can be easily identified
* Create new section at the very end of this list "Legacy SAFE Template (v1)" and put all these pages there?